config/config.yaml

```
@@ -78,6 +78,10 @@ storage:
     # If null - allow unlimited transfers.
     #outgoing_shard_transfers_limit: 1
 
+    # Enable async scorer which uses io_uring instead of mmap based backend.
+    # Before using, read io_uring documentation on specific tuning cases. 
+	  #async_scorer: true
```

```suggestion
    # Enable async scorer which uses io_uring when rescoring.
    # Before using, read io_uring documentation on specific tuning cases. 
    #async_scorer: true
```

or even:

```suggestion
    # Enable async scorer which uses io_uring when rescoring.
    # Only supported on Linux, must be enabled in your kernel.
    # See: <https://qdrant.tech/articles/io_uring/#and-what-about-qdrant>
    #async_scorer: true
```

@generall may have a better description.

---

lgtm